app/Jobs/Emails/Traits/SummitEmailJob.php

```diff
@@ -24,8 +24,6 @@ trait SummitEmailJob
      * @return array
      */
     public function emitSummitTemplateVars(array $payload, Summit $summit):array{
-        $payload[IMailTemplatesConstants::summit_name] = $summit->getName();
```

Comment:
all this could be moved to new AbstractSummitEmailJob class ?

Reply:
done